--- taskjuggler-2.4.1/configure.in.orig	2008-05-06 03:04:55.000000000 -1000
+++ taskjuggler-2.4.1/configure.in	2008-11-18 20:20:44.000000000 -1000
@@ -101,9 +101,9 @@
     LIBKCAL="-lkcal"
     dnl Some distros split off a KDEPIM devel package most people
     dnl forget to install.
-    KDE_CHECK_HEADERS(libkcal/calendarlocal.h libkcal/todo.h)
-    if test "$ac_cv_header_libkcal_calendarlocal_h" = "no" -o \
-            "$ac_cv_header_libkcal_todo_h" = "no" ; then
+    KDE_CHECK_HEADERS(kcal/calendarlocal.h kcal/todo.h)
+    if test "$ac_cv_header_kcal_calendarlocal_h" = "no" -o \
+            "$ac_cv_header_kcal_todo_h" = "no" ; then
       AC_MSG_ERROR([KDEPIM header files no found. Please make sure
 you have KDEPIM and the corresponding development packages
 installed!])
--- taskjuggler-2.4.1/taskjuggler/ICalReport.h.orig	2008-05-06 03:04:54.000000000 -1000
+++ taskjuggler-2.4.1/taskjuggler/ICalReport.h	2008-11-18 20:34:40.000000000 -1000
@@ -13,8 +13,8 @@
 #ifndef _ICalReport_h_
 #define _ICalReport_h_
 
-#include <libkcal/calendarlocal.h>
-#include <libkcal/todo.h>
+#include <kcal/calendarlocal.h>
+#include <kcal/todo.h>
 
 #include <Report.h>
 
--- taskjuggler-2.4.1/taskjuggler/ICalReport.cpp.orig	2008-05-06 03:04:54.000000000 -1000
+++ taskjuggler-2.4.1/taskjuggler/ICalReport.cpp	2008-11-18 20:34:36.000000000 -1000
@@ -20,7 +20,7 @@
 #include <qptrdict.h>
 #include <qfile.h>
 #include <klocale.h>
-#include <libkcal/icalformat.h>
+#include <kcal/icalformat.h>
 
 #include "tjlib-internal.h"
 #include "Project.h"
