--- kdepim-3.4.0/kandy/src/Makefile.am.tn	2005-03-23 17:17:21.236167534 +0100
+++ kdepim-3.4.0/kandy/src/Makefile.am	2005-03-23 17:19:15.397283296 +0100
@@ -30,6 +30,8 @@
 
 kde_kcfg_DATA = kandy.kcfg
 
+KDE_ICON = kandy
+
 install-data-local: uninstall.desktop
 	$(mkinstalldirs) $(DESTDIR)$(kde_appsdir)/Utilities
 	$(INSTALL_DATA) $(srcdir)/uninstall.desktop $(DESTDIR)$(kde_appsdir)/Utilities/kandy.desktop
